app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from "react";
+
+import { Formik, Form as FormikForm } from "formik";
+import { Button, Pane, Toastr } from "neetoui";
+import { Input, Select, Textarea } from "neetoui/formik";
+
+import { ROLES } from "../constants";
+
+const Form = ({ onClose, contact }) => {
+  const [submitted, setSubmitted] = useState(false);
+
+  const handleSubmit = () => {
+    setSubmitted(true);
+    Toastr.success("Contact added successfully.");
+    onClose();
+  };
+
+  return (
+    <Formik
+      initialValues={contact}
+      validateOnBlur={submitted}
+      validateOnChange={submitted}
+      onSubmit={handleSubmit}
+    >
```

Comment:
Where is the validation schema?

Reply:
I created the Contacts listing component and wanted the default format for Form. I worked on validation schema in the new contact component that's why I removed it from here.